en/guide/serialization.md

```diff
@@ -128,8 +126,12 @@ The reordering happens as follows:
 
 ### MAVLink 2 Field Reordering
 
-MAVLink 2 messages order the MAVLink 1 ("base") fields in the same way as the MAVLink 1 protocol. 
-Extension fields in MAVLink 1 messages, and all fields in new MAVLink 2 messages (id>255), are ordered in the same way as the source XML.
+For messages with ids in the MAVLink 2 range (id >255), all fields are ordered in the same way as MAVLink 1 above.
```

Comment:
This is just creating confusion. All primary message fields (IE anything before an `extension` tag) will be reordered per the ordering rules, anything that is an extension will not be reordered. There is no special handling related to `id > 255` I'd suggest that this section just be titled `MAVLink 2 Extensions Fields` or something and highlight that anything that is an extension will not be subject to above reordering, and will not be included in the CRC.

Reply:
Upon further review the reordering appears to be touched on in 3 documents here mavlink2.md, serilization.md, and overview.md. I'd suggest that we constrain the technical details to one of these documents and link from the others, rather then trying to maintain the description in 3 places. (It's probably mostly a serialization issue and can be shrunk in the other fields)

Reply:
Very concise description - @hamishwillee that's what I was trying to get across in our discussion earlier.

Reply:
@WickedShell Yes to both. I've simplified in https://github.com/mavlink/mavlink-devguide/pull/100/commits/3f97760e9dc5c370b930990d2e468463367e6c43

Appreciate your thoughts.